lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -19,6 +19,28 @@ has version => 2;
 
 sub E { JSON::Validator::Error->new(@_) }
 
+has generate_definitions_path => sub {
+  my $self = shift;
+  Scalar::Util::weaken($self);
+
+  return sub {
+    my $ref = shift;
+    my @path;
+
+    # Try to determin the path from the fqn
+    if ($ref->fqn =~ m|^(.*)#/(.+)$|) {
+      my $relative = $1;
+      my $path     = $2;
+
+      @path = split '/', $path;
+      pop @path;
+    }
+
+    if   (@path) { \@path }
+    else         { [$self->{definitions_key} || 'definitions'] }
```

Comment:
`$self->{definitions_key}` should not be used, since it's private to JSON::Validator.

Also, can you change it to:

```
return @path ? \@path : ['definitions'];
```

I don't think I have to ask you to run the code through perltidy, if you do that ;)

Reply:
fixed in upcoming push, and off course, tidy is implied :)